performance_manager/lib/gtfs_static_table.py

```diff
@@ -0,0 +1,317 @@
+from typing import List, Optional
+
+import logging
+import pandas
+import numpy
+
+import sqlalchemy as sa
+from sqlalchemy.orm import sessionmaker
+
+from .s3_utils import read_parquet, file_list_from_s3
+from .postgres_utils import MetadataLog, get_unprocessed_files
+from .postgres_schema import (
+    StaticFeedInfo,
+    StaticTrips,
+    StaticRoutes,
+    StaticStops,
+    StaticStopTimes,
+    StaticCalendar,
+)
+from .gtfs_utils import start_time_to_seconds
+
+
+def get_static_parquet_paths(table_type: str, feed_info_path: str) -> List[str]:
+    """
+    get static table parquet files from FEED_INFO path
+    """
+    springboard_bucket = "mbta-ctd-dataplatform-dev-springboard"
```

Comment:
hard coded bucket name, this should be handled with .env variable...

Reply:
this is currently in our .env file which should be loaded when launchgin.

use `EXPORT_BUCKET` for now. eventually we should standardize our bucket names across everything.

https://app.asana.com/0/1189492770004753/1202862782004919/f